Bottom: c2ca8b437afa6a8f4b310e1dacc427fdb33bbd95
Top:    e5b87b9f49fcd5500dd2da50a8f71c5047630787
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-18 20:47:01 +0000

Batch size for accumulating lines down to 1

This works around a bug seen on one of my boards, and is similar
to what I had in the non-pours variant of the code.


---

diff --git a/src/pour.c b/src/pour.c
index d1099cd..d9ee86f 100644
--- a/src/pour.c
+++ b/src/pour.c
@@ -68,7 +68,7 @@ RCSID ("$Id$");
 
 #define UNSUBTRACT_BLOAT 100
 #define SUBTRACT_PIN_VIA_BATCH_SIZE 100
-#define SUBTRACT_LINE_BATCH_SIZE 20
+#define SUBTRACT_LINE_BATCH_SIZE 1
 
 /* ---------------------------------------------------------------------------
  * local prototypes
